bot/constants.py

```diff
@@ -0,0 +1 @@
+token = ''
```

Comment:
This is not going to work. We can't be committing the token into the source control, so we'll need to find a better solution for this. Perhaps an environment variable?

Reply:
Also, for any constants we do need, perhaps dataclasses (or NamedTuples) would be a more organized option. Look at how we do it in Python for examples on how this might work.

Reply:
A temporary solution is a environ.get to get the bot token from a env. variable.